datacenter/dtr/2.2/guides/admin/configure/job-runner.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Jobrunner
+description: Learn about the inner-workings of the jobrunner container in the DTR workflow.
+keywords: docker, job, runner
+---
+
+The jobrunner is presented as a container in DTR. It is a DTR mechanism that:
+
+1. Consumes jobs from a universal jobs queue 
+2. Performs the work of the given action
+
+Having multiple replicas in your cluster increases the number of jobrunner 
+containers, which will increase the number of concurrent jobs.
+
+## Jobrunner Workflow
+The workflow of the jobrunner can be depicted as follows:
```

Comment:
"can be depicted" sounds like you are not sure if the diagram is accurate. Just say "the following diagram depicts the behavior of the jobrunner"